src/main/java/frc/robot/subsystems/PneumaticShooterSubsystem.java

```diff
@@ -17,24 +17,37 @@
   // Put methods for controlling this subsystem
   // here. Call these from Commands.
 
-  private final DoubleSolenoid doubleSol = new DoubleSolenoid(0,1);
+  private final DoubleSolenoid doubleSolenoid = new DoubleSolenoid(0,1);
```

Comment:
why did you change the name to doubleSolenoid? doubleSol is shorter and cooler

Reply:
no

Reply:
idk, i was lazy, I called climb solenoid double solenoid2. I

Reply:
: (

Reply:
pls just accept